client/src/pages/DuelCreate.jsx

```diff
@@ -12,11 +12,35 @@ import PageError from "../components/status/PageError";
 const DuelCreate = ({ history }) => {
   const [searchRegex, setSearchRegex] = useState(null);
   const [selected, setSelected] = useState(null);
+  const [search, setSearch] = useState(null);
   const [error, setError] = useState(null);
   const queryClient = useQueryClient();
 
   const { data: listOfUsers, isSuccess, isError } = useQuery(["users", "challengeable"]);
 
+  const [system, setSystem] = useState(0);
+  const [difficulty, setDifficulty] = useState(1);
+  const [dataSystems, setData] = useState([]);
+
+  const changeSystem = (event) => setSystem(event.target.value);
+  const changeDifficulty = (event) => setDifficulty(event.target.value);
+
+  useEffect(() => {
+    async function createData() {
+      const result = await axios.get(`/api/v1/chemicals/systems`);
+      setData(result.data);
+    }
+    createData();
+  }, []);
```

Comment:
Pourquoi créer une fonction ici ? 
```suggestion
  useEffect(() => {
      const result = await axios.get(`/api/v1/chemicals/systems`);
      setData(result.data);
  }, []);
```